.github/workflows/test-rc-release.yaml

```diff
@@ -170,7 +170,7 @@ jobs:
           git add setup.cfg
           git commit -m "Updating setup.cfg with RC provider packages on $rc_issue_url"
           git push origin ${{ steps.create_rc_branch.outputs.rc_testing_branch }}
-          gh pr create --base ${{ steps.create_rc_branch.outputs.rc_testing_branch }} \
+          gh pr create --base main \
```

Comment:
Do we need to specify base explicitly as I think it's our default branch and will be used as base anyway, no? If that is the case we may not need to specify --base argument.

Reply:
yes, I think we can ignore it. let me remove it.

Reply:
just removed